ctlra/devices/ni_maschine_mk3.c

```diff
@@ -721,18 +721,18 @@ ni_maschine_mk3_usb_read_cb(struct ctlra_dev_t *base,
 		}
 
 		/* Main Encoder */
-		struct ctlra_event_t event = {
-			.type = CTLRA_EVENT_ENCODER,
-			.encoder = {
-				.id = 0,
-				.flags = CTLRA_EVENT_ENCODER_FLAG_INT,
-				.delta = 0,
-			},
-		};
-		struct ctlra_event_t *e = {&event};
```

Comment:
Is there a reason to move this to below?

Reply:
I think its not needed to initialize it here, if this is false: https://github.com/openAVproductions/openAV-Ctlra/pull/107/files#diff-8e572deadc8b71dc3a2aa7237af81c0e65155cb06dc3561dbfa445ffd7751164L734

Reply:
Ah yes - Ok - so its not a bugfix or something, just an improvement. Typically I'd try keep it in a different commit, but its only minor so lets merge this thing!